.claude/skills/vault-lint-test/scripts/run.sh

```
@@ -0,0 +1,169 @@
+done
+
+if ! [[ "$START_AT" =~ ^[1-5]$ ]]; then
+    echo "--from must be 1..5, got: $START_AT" >&2
+    exit 2
+fi
+
+REPO_ROOT="$(git rev-parse --show-toplevel 2>/dev/null)" || {
+    echo "Not in a git repository." >&2
+    exit 2
+}
+cd "$REPO_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)